src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -583,7 +583,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 <Target Name="_GetReferenceAssemblyPaths">
 	<GetReferenceAssemblyPaths
 			TargetFrameworkMoniker="$(TargetFrameworkIdentifier),Version=v1.0"
-			RootPath="$(TargetFrameworkRootPath)">
+			RootPath="$(XATargetFrameworkRootPath)">
```

Comment:
Where is the value for `$(XATargetFrameworkRootPath)` defined? I can't see it in x-a or monodroid. How does `GetReferenceAssemblyPaths` behave if that is empty? Should it have a default value of `$(TargetFrameworkRootPath)` if it is not set?